sg_jira/jira_session.py

```diff
@@ -509,26 +510,48 @@ def create_issue_from_data(self, jira_project, issue_type, data):
         # It seems a Project `simplified` key can help distinguish between old
         # school projects and new simpler projects.
         # TODO: cache the retrieved data to avoid multiple requests to the server
-        create_meta_data = self.createmeta(
-            jira_project,
-            issuetypeIds=jira_issue_type.id,
-            expand="projects.issuetypes.fields",
-        )
-        # We asked for a single project / single issue type, so we can just pick
-        # the first entry, if it exists.
-        if (
-            not create_meta_data["projects"]
-            or not create_meta_data["projects"][0]["issuetypes"]
-        ):
-            logger.debug(
-                "Create meta data for Project %s Issue type %s: %s"
-                % (jira_project, jira_issue_type.id, create_meta_data)
+
+        if self._is_jira_cloud or self._version < (9, 0, 0):
```

Comment:
@julien-lang Answering your question:

> what happens to people using an old Jira version, prior to this deprecation?

I've added this conditional so older versions of Jira Server can use the existing logic.

Reply:
Thanks, that's good!